src/lib/toolbar/toolbar.ts

```diff
@@ -44,10 +39,35 @@ export const _MdToolbarMixinBase = mixinColor(MdToolbarBase);
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None
 })
-export class MdToolbar extends _MdToolbarMixinBase implements CanColor {
+export class MdToolbar extends _MdToolbarMixinBase implements CanColor, AfterViewInit {
 
-  constructor(renderer: Renderer2, elementRef: ElementRef) {
+  /** Element reference to the default first row of the toolbar. */
+  @ViewChild('defaultFirstRow', { read: ElementRef }) _defaultFirstRow: ElementRef;
+
+  constructor(renderer: Renderer2, elementRef: ElementRef, private _platform: Platform) {
     super(renderer, elementRef);
   }
 
+  ngAfterViewInit() {
+    // Do nothing if we're not on the browser platform.
+    if (!this._platform.isBrowser) {
```

Comment:
If you abort here on the server, the prerendered toolbar probably won't look right